src/styles/theme.css

```diff
@@ -1,8 +1,38 @@
-body {
-  color: green;
-}
-
 h1, h2, h3, h4, h5, h6 {
   text-align: center;
-  color: black;
+}
+
+@font-face {
+  font-family: "Geist-Sans";
+  src: url('../../node_modules/geist/dist/fonts/geist-sans/Geist-Regular.woff2') format("truetype");
+}
+
+@font-face {
+  font-family: "Geist-Mono";
+  src: url('../../node_modules/geist/dist/fonts/geist-mono/GeistMono-Regular.woff2') format("truetype");
+}
+
+* {
+  font-family: 'Geist-Sans', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+:root {
+  --color-primary: #016341;
+  --color-secondary: #0E0F0C;
+  --color-tertiary: #151818;
+  --color-accent: #C3DDBA;
+  --color-gray: #F5F5F5;
+  --color-gray-inner: #EDEFEC;
+  --color-header: #0E0F0C;
+  --color-text: #4D4D4D;
+  --color-white: #FFFFFF;
+  --font-primary: 'Geist', 'sans-serif';
+  --font-size-base: 16px;
+  --container-radius: 6rem;
```

Comment:
similar to my comments above, we don't have to capture _page / component_ specific tokens for the design system, we can start with just the ones that apply to broadest parts of the design that should be used everywhere, like colors and fonts.  This can always grow as the site grows.